### PR TITLE
Storybook rollout

### PR DESCRIPTION
**Pull request by ghost**
This depends on #32 

This really just adds storybook and puts ActionTooltip in there (ActionFetch pending other fixes). Whenever you do `yarn storybook`, it will now bring up the storybook interface. In theory, as we develop other tooltip displays, we should also make storybook analogues for testing / design vetting.

* Add storybook
* Remove storybook default examples
* Add ActionTooltip example

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +34,9 @@
     "lint": "eslint --ext .js --ext .vue src",
     "docs": "vue-docgen -c docgen.config.js",
     "build": "vue-cli-service build --target lib --formats umd-min --name VueXIVTooltips --filename vue-xivtooltips src/index.js",
-    "build:dev": "vue-cli-service build --mode development --target lib --formats umd --name VueXIVTooltips --filename vue-xivtooltips src/index.js"
+    "build:dev": "vue-cli-service build --mode development --target lib --formats umd --name VueXIVTooltips --filename vue-xivtooltips src/index.js",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```
Comment (ayyaruq) [inline package.json:39]: Are these 2 actual tasks defined somewhere, like does storybook CLI add the commands?

**Review by ayyaruq — COMMENTED** [review]
Dunno about example but just that one question. Otherwise looks fine.

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json:39]: the storybook setup adds those commands, but they do work. I forgot to add a babel package removed during the cleanup, but ultimately not terrible.

Comment (ayyaruq) [reply package.json:39]: Just for clarity, these get installed to `node_modules/.bin` which yarn scripts have in their exec path.

**Pull request merged**